timescaledb/getting-started/create-cagg/create-cagg-basics.md

```diff
@@ -94,6 +94,7 @@ SELECT * FROM timescaledb_information.continuous_aggregates;
 
 Results:
 
+```bash
```

Comment:
Is this supposed to be here? The table won't render if it's inside a code block.